starwars/src/components/Navigation.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react'
+
+export default class Navigation extends Component {
+    render() {
+        let classLeft='',classRight=''
+        if(this.props.init>1)
+            classLeft='available'
+        if(this.props.init%2===1)
+            classRight='available'
+        return (
+            <nav>
+                <section
+                    onClick={this.props.prev}
+                    className={classLeft}
+                >
+                    {this.props.init>1 && "Previous"}
+                </section>
+                <section
+                    onClick={this.props.next}
+                    className={classRight}

```

Comment:
Great job building the logic here even accounting for if there are any pages left that you can ask the api for. I also like that you separated this into a sub-component and simply plug it in where needed in your app. This is what I mean when I say composing a react application of sub-components!